docs/usage.md

```diff
@@ -0,0 +1,219 @@
+# Usage
+
+Lrud handles the registering of new navigation nodes, the handling of key events, and the emitting of events based on internal state changes.
+
+For some examples of common desired behaviour and how to achieve it, see our [common recipes](./recipes.md).
+
+## Creating a new instance
+
+```js
+const { Lrud } = require('Lrud')
+
+const navigation = new Lrud()
+```
+
+## Registering a node
+
+A node can be added to the navigation tree by calling `navigation.registerNode` with the `id` of the node and any node options
+
+```js
+navigation.registerNode('root', {})
+```
+
+The very first node that is registered will become the *root node*. From here, any node that is registered without a parent will automatically become a child of the root node.
+
+```js
+navigation
+    .registerNode('root', { orientation: 'horizontal' })
+    .registerNode('alpha', { isFocusable: true })
+
+// ...is the same as
+
+navigation
+    .registerNode('root', { orientation: 'horizontal' })
+    .registerNode('alpha', { isFocusable: true, parent: 'root' })
+```
+
+## Registering options
+
+Most options affect behaviour for when Lrud is handling key events and assigning focus.
+
+### `orientation`
+
+`"vertical" | "horizontal"`
+
+Any node that has children should be registered with an orientation. A node of vertical orientation will handle up/down key events while horizontal will handle left/right key events.
+
+### `isWrapping`
+
+`boolean`
+
+Used in conjunction with orientation to make a list wrap at the top/bottom or left/right depending on orientation.
+
+### `index`
+
+`number`
+
+The index of the node, relative to its siblings. If no idex is given, it will be set as the _next_ index under its parent.
+
+```js
+navigation
+    .registerNode('X', { parent: 'root' })
+    .registerNode('Y', { parent: 'root' })
+    .registerNode('Z', { parent: 'root' })
+
+// ...is the same as
+
+navigation
+    .registerNode('X', { parent: 'root', index: 0 })
+    .registerNode('Y', { parent: 'root', index: 1 })
+    .registerNode('Z', { parent: 'root', index: 2 })
+```
+
+### `isIndexAlign`
+
+`boolean`
+
+To be used in conjunction with orientation to give a node index alignment functionality.
+
+Index alignment ensures that when focus would move from one this nodes descendants to another, Lrud will attempt to ensure that the focused node matches the index of the node that was left.
```

Comment:
Sorry, just little things... "one of this" here

Reply:
No apologies needed - English are hard.